### ISSUE TITLE
List suggested companies for more countries

### ISSUE THREAD

**Issue by baltpeter** [datenanfragen/data#230]
With our suggested companies lists, we collect companies that have data on most people in their respected countries. Prominent examples for such companies include credit agencies and list brokers. We recommend sending GDPR access requests to all these companies.  
This explicitly doesn't include companies that are relevant for many but not all users (like Google, Facebook, Amazon etc.)

We currently have suggested company lists for Austria, Germany and the UK. We would like to support more countries. All EU countries are fine.

The lists are found in the [`suggested-companies` directory](https://github.com/datenanfragen/data/tree/master/suggested-companies). There is a JSON array of the suggested company slugs for each country. (The slug is simply our [internal ID for each record](https://github.com/datenanfragen/data/tree/master/companies). As a result, a company needs to be in our database before it can be suggested.)

**Comment by ManishAradwad** [comment]
Hey!! I'd love to work on this one...

**Comment by baltpeter** [comment]
Sure, thank you! Which country would you like to do?

Let me know if you have any questions.

**Comment by tstrohmeier** [comment]
@baltpeter I can help with Austria.

**Comment by zner0L** [comment]
> @baltpeter I can help with Austria.

You already contributed a list. Are you thinking about expanding it? You are very welcome to do so btw :)

**Comment by ManishAradwad** [comment]
I'll go for Denmark...
Just a question,
I'll have to fetch all those companies from database with relevant country as Denmark, right?

**Comment by baltpeter** [comment]
I don't think we currently have any Danish companies in the database, so they would first have to be added.

**Comment by achoudh5** [comment]
@baltpeter I see that suggested companies are already present in the list of companies folder. Can you please reiterate what exactly needs to be done, I am a little confused

**Comment by baltpeter** [comment]
We already have suggested companies for Austria, Germany and the UK in [this folder](https://github.com/datenanfragen/data/tree/master/suggested-companies). We want to expand the list for more EU countries.

The first step is compiling a list of those suggested companies for the new country in question. Then, you need to check for each one if it is already present in the database or not. If not, it needs to be added first.

**Comment by baltpeter** [comment]
The JSON array of slugs for each country is a list of references to the existing records in the `companies` folder.

**Comment by tstrohmeier** [comment]
> > @baltpeter I can help with Austria.
> 
> You already contributed a list. Are you thinking about expanding it? You are very welcome to do so btw :)

I know ;) I can check if the list is complete.

Included should be all major companies of a country right?

**Comment by zner0L** [comment]
> Included should be all major companies of a country right?
 Well, we are especially interested in those collecting significant amounts of data that not everybody might know about. Like credit agencies and such. Those are also pretty country specific. So the companies do not have to be major in all fields, just companies that you suspect to collect data on almost everyone.

**Comment by ManishAradwad** [comment]
> I don't think we currently have any Danish companies in the database, so they would first have to be added.

Ok then I'll start fetching some relevant Danish companies,
Can you suggest some sources where I can find information need to be added

**Comment by tstrohmeier** [comment]
> > Included should be all major companies of a country right?
> 
> Well, we are especially interested in those collecting significant amounts of data that not everybody might know about. Like credit agencies and such. Those are also pretty country specific. So the companies do not have to be major in all fields, just companies that you suspect to collect data on almost everyone.

To get the same standards for all EU countries, it would be maybe a good idea to write a guide with a list of line of businesses which usual collect data on everyone.

**Comment by baltpeter** [comment]
> Ok then I'll start fetching some relevant Danish companies,
> Can you suggest some sources where I can find information need to be added

@ManishAradwad As for a list of relevant companies, no sorry. The whole point of this issue is that we don't have that information for countries other than Germany and are looking for people who do, after all. :)

As for finding the relevant data on a particular company, their privacy policy usually contains all of that. In addition, I have mentioned some more good sources in the README of this repository.

**Comment by baltpeter** [comment]
> To get the same standards for all EU countries, it would be maybe a good idea to write a guide with a list of line of businesses which usual collect data on everyone.

@tstrohmeier I *tried* going in that direction with [this section of the README](https://github.com/datenanfragen/data#data-formats). I agree that this is not very comprehensive or clear, though. My main issue here is that I have a very Germany-centric view. I don't really know the situation in other countries, so I have not been able to establish clear guidelines here.  
Currently, the list boils down to the following categories:

* credit agencies
* list/address brokers (including email marketing)
* governmental agencies (local residents' offices, departments of transportation, tax offices etc.) — even our German list is currently lacking in this regard, the problem is that there is usually not a single nationwide 'instance' of these entities but rather local ones at the state or even city level
* (maybe) known spam companies, although I am not too sure about that as that would put us in a position of deciding who falls into that category which I don't want

If you or anyone else has any suggestions for changes, I'd be happy to work on an official 'policy' for the suggested companies. We should maybe start a separate issue for that.

**Comment by mrhspecter** [comment]
Would take a shot at Slovenia if you thats okay!

**Comment by baltpeter** [comment]
@mrhspecter Absolutely, thank you!

**Comment by mathagician** [comment]
@baltpeter I would like to work for India. I understand that I should add a new file as per the slug in the `company` directory with its content in the JSON form? Can you please guide?

**Comment by zner0L** [comment]
> @baltpeter I would like to work for India. I understand that I should add a new file as per the slug in the `company` directory with its content in the JSON form? Can you please guide?

Since the GDPR is not valid for citizens of India, I am afraid that might be useless. We only support suggested countries within the European Economic Area, I am sorry. You are, however, very welcome to develop a clone of datenanfragen to use with Indian privacy laws and I'd be happy to support you with that.

**Comment by mathagician** [comment]
@baltpeter are only fincance companies required like credit rating companies? Can a food company be suggested?

**Comment by baltpeter** [comment]
@mathagician The idea is to collect companies that have data on most citizens of a particular country, as well as those that collect data on lots of people without these people necessarily knowing.

What do you mean by a food company? If they only collect data on their customers, we probably wouldn't suggest them. Or do you know a food company that would be relevant for most people in a country?

**Comment by gioboa** [comment]
I will go for Italy for hacktoberfest 2020, if for you it's ok.

**Comment by baltpeter** [comment]
@gioboa Great, thanks!

**Comment by kzuri** [comment]
Heyy @baltpeter, I think I would expand list further. Will update you if I think to add new country to DB

**Comment by Stig124** [comment]
I'll start adding French companies
